specs/StartInBackgroundMode.md

```diff
@@ -0,0 +1,101 @@
+CreationPriority
+===
+
+# Background
+WebView2 has a functionality enabled by default which raises the launch priority of WV2
+processes to ensure that WV2 processes are given higher CPU priority and access to resources 
+while creation, and that can bring improvements on time spent creating a WV2 instance. Since 
+this improvement is at the cost of taking higher CPU priority, we introduce `CreationPriority` 
+property to give developers an option to decide by themselves.
+
+# Description
+* Setting `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_NORMAL` will signal WebView2 
+to use less system resources during the creation. 
+* Setting `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_HIGH` will attempt to enable 
+the launch time improvement functionality with raising the launch priority during the creation.  
+* Default value is `COREWEBVIEW2_CREATION_PRIORITY_HIGH`.
+## Notes
+* The host app's priority is at least normal before applying the launch time improvement functionality. 
+Else, WebView2 still get created with normal priority even with `COREWEBVIEW2_CREATION_PRIORITY_HIGH` 
+setting.
+* Currently launch time improvement functionality is only enabled for WebView2 manager process, so the 
+`CreationPriority` only effects the creation of manager process. `CreationPriority` can play the same 
+role for other process types like render process if WebView2 opts in launch time improvement functionality 
+for those types in the future.
+# Examples
+## Win32 C++
+```cpp
+auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+// Set `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_NORMAL` if you don't want to boost the 
+// launch time at the cost of using higher CPU priority during the launch phase.
+options->put_CreationPriority(COREWEBVIEW2_CREATION_PRIORITY_NORMAL);

```

Comment:
Why would an end developer want to have normal pri for their WebView2 creation? Can we work the answer to that question into the sample code? That is, can the sample code show why we're setting the creation priority? (If not, perhaps just adding a comment about **why** we're setting normal pri in the sample code)

Reply:
> Why would an end developer want to have normal pri for their WebView2 creation?

The use case for normal pri is end developer does not want to receive the higher CPU pri.

>  That is, can the sample code show why we're setting the creation priority?

No, I don't think so. 

> If not, perhaps just adding a comment about why we're setting normal pri in the sample code

What about "Set `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_NORMAL` to prevent applying the higher CPU priority during WebView2 browser process Creation".?

Reply:
> The use case for normal pri is end developer does not want to receive the higher CPU pri.

But why don't they want higher CPU priority?